src/odemis/dataio/tiff.py

```diff
@@ -2359,9 +2368,14 @@ def _getOMEXML(self, tfile):
         tfile.SetDirectory(0)
         desc = tfile.GetField(T.TIFFTAG_IMAGEDESCRIPTION)
 
-        if (desc and ((desc.startswith(b"<?xml") and b"<ome " in desc.lower()) or
-                      desc[:4].lower() == b'<ome')):
+        if (desc and ((b"<?xml" in desc) and b"<ome " in desc.lower() or
+                desc[:4].lower() == b'<ome')):
             try:
+                pattern = b'<?xml version="1.0" encoding="UTF-8"?>'
```

Comment:
Be more flexible in the string you search. I think that just "<?xml" is enough. Maybe you could also be more strict, in checking that it's the first word of a line. Also, if it starts just with "<ome" that will not work...